proto/v1/setting_service.proto

```diff
@@ -61,50 +62,11 @@ message Value {
   oneof value {
     // Defines this value as being a string value.
     string string_value = 1;
-    SMTPMailDeliverySettingValue smtp_mail_delivery_setting_value = 2;
+    store.SMTPMailDeliverySetting smtp_mail_delivery_setting_value = 2;
```

Comment:
I prefer to decouple the store proto and api proto, although they are mostly the same

Reply:
True it's another good way to separate the API message and store message. But as we did have the store message, and for many messages like policy payload and setting payload, they're just share the same struct so I'd like to only use one message to save the effort until we have to separate them.